apisix/core/json.lua

```diff
@@ -95,8 +100,19 @@ local delay_tab = setmetatable({data = "", force = false}, {
 })
 
 
--- this is a non-thread safe implementation
--- it works well with log, eg: log.info(..., json.delay_encode({...}))
+---
+-- Delayed encoding of input data, avoid unnecessary encode operations.
+-- This is a non-thread safe implementation, it works well with log.
```

Comment:
I think we can avoid the non-thread description, as most of the API is not thread safe.

Reply:
update